api/config/log_edit_email.php

```diff
@@ -0,0 +1,11 @@
+<?php
```

Comment:
@jalmiShreya lets get rid of this since we don't need it any more as config values come from DB.